modules/microshift-auto-apply-manifests.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-auto-apply-manifests_{context}"]
+= Auto-applying manifests
+
+When you run `kustomize`, {product-title} searches `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k command` to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-name} workloads.
+
+[IMPORTANT]
+====
+`Kustomize` must be either running as a separate step in the boot process, or it must be part of the {product-title} image.
```

Comment:
Again, guessing at "command", a different noun might be better here, but just to illustrate the format:
```suggestion
The `kustomize` command must be either running as a separate step in the boot process, or it must be part of the {product-title} image.
```